content/en/docs/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -110,6 +121,24 @@ kfctl apply -V -f ${CONFIG_FILE}
 
 The Kubeflow endpoint is exposed with [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) 31380. If you don't have any experience on Kubernetes, you can [expose the Kubeflow endpoint as a LoadBalancer](#expose-the-kubeflow-endpoint-as-loadbalancer) and access the **EXTERNAL_IP**.
 
+### Extra network setup requirement for VPC-Gen2 Clusters only.
+
+**NOTE** These steps are not required for Classic clusters, i.e. where `WORKER_NODE_PROVIDER` is set to `classic`.
+
+A VPC gen2 cluster does not assign a public ip address to the kubernetes master node by default.
+It provides access via a Load balancer, which is configured to allow only a set of ports over public internet.
+So to access the cluster's resources in VPC gen2 cluster, following steps are required.
+
+* Option 1: Configure via Load balancer. Follow: [expose the Kubeflow endpoint as a LoadBalancer](#expose-the-kubeflow-endpoint-as-loadbalancer)
+
+* Option 2: If you need access to nodes or NodePort in the VPC, this can be achieved by starting another instance in the 
+same VPC and assigning it a public ip(i.e. floating ip). Now either by ssh login to the instance or by creating a ssh socks proxy.
+
+for example:
+`ssh -D9999 root@new-instance-public-ip `
+
+And then configure socks proxy at `localhost:9999` and access cluster services.
```

Comment:
Please capitalize letter where appropriate

Suggestion

```suggestion
* Option 2: If you need access to nodes or NodePort in the VPC, this can be achieved by starting another instance in the 
same VPC and assigning it a public IP (i.e. the floating IP). Next, use SSH to log into the instance or create an SSH socks proxy, such as `ssh -D9999 root@new-instance-public-ip`.

Then, configure the socks proxy at `localhost:9999` and access cluster services.
```